tests/main/test_launch.py

```diff
@@ -293,7 +293,10 @@ def test_main_entrypoint__correctly_assigns_shared_values_dictionary_to_process_
     object_access_dict = app_info["object_access_inside_main"]
     shared_values_dict = object_access_dict["values_to_share_to_server"]
     test_process_monitor = object_access_dict["process_monitor"]
-    assert test_process_monitor._values_to_share_to_server is shared_values_dict
+    assert (
+        test_process_monitor._values_to_share_to_server  # pylint: disable=protected-access
```

Comment:
I don't think we're using pylint anymore, is there any reason for this comment?

Reply:
I don't remember adding this. I think it might be a product of the pre-commits I ran. I will remove it